pkg/zson/docs/spec.md

```diff
@@ -144,7 +140,7 @@ Here is a pseudo-grammar for zson types:
 
 <column> := <id> : <type>
 
-<id> := <identifier as defined by javascript spec>
+<id> := <identifier as defined by JavaScript spec>
```

Comment:
Can someone point to the ideal URL we could reference for this specific bit of JavaScript spec? A generic web search yields tons of hints on various forums/etc. but to make the spec complete it would seem ideal to point to a definitive one.

Reply:
I think https://tc39.es/ecma262/#prod-IdentifierName is as good as it gets.

Reply:
@aswan will add something here.